tembo-operator/src/dedicated_networking.rs

```diff
@@ -427,8 +421,23 @@ async fn reconcile_dedicated_networking_service(
     service_spec.insert("type".to_string(), json!(service_type));
     let ip_allow_list = cdb.spec.ip_allow_list.clone().unwrap_or_else(|| vec![]);
 
-    if service_type == "LoadBalancer" && !ip_allow_list.is_empty() {
-        service_spec.insert("loadBalancerSourceRanges".to_string(), json!(ip_allow_list));
+    // Allow ip_allow_list to allow all entries are in CIDR notation
```

Comment:
Were you able to test this locally? If not please do that in dev.

Reply:
Tested it locally. Will test it out again in dev once its merged